### PR TITLE
adds robust addon support

### PR DESCRIPTION
**Pull request by CNDW**
addon support take 2. Now that we are leveraging ember's module loader, addons are possible.

This wound up being a lot messier than I had originally thought it would be, but it works pretty well. There shouldn't be any backwards compatibility breakage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +139,12 @@ For ease of use with npm scripts and for quick testing.
     -p, --pod-module-prefix [pod-module-prefix]  Pod Module Prefix, the directroy that the ember-resolver uses for pods
 ```
 
+### Ember Addons
+
+There is basic support for ember-addons. You should be able to npm install the addon, and assuming the published addon conforms to ember's addon standard, emberate will include the addon in your app bundle entry file.
+
+You can optionally exclude the addon support from your build by setting `addonSupport` to false in the emberate options. Since Ember Addons are normally written in es6, you will need to include a transpiler in your browserify bundle. The most simple way to do so is with the [babelify transform](https://github.com/babel/babelify).
+
```
Comment (knownasilya) [inline README.md]: Maybe worth noting that default blueprints would have to be run manually, not sure if you've thought of that.

Comment (CNDW) [inline README.md]: I thought blueprints where for ember-cli? Maybe I'm misunderstanding what the blueprints are...

File: lib/util/copyAddonFile.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+'use strict';
+/**
+ *  Ember addons are often written in a way that is incompatable
```
Comment (knownasilya) [inline lib/util/copyAddonFile.js]: typo: incompatable

**Comment by CNDW** [comment]
@knownasilya where there any other issues with this work? Or is this good to go? One thing I have been thinking is to change the default `addonSupport` to false to prevent any unwanted side effects and keep this as a minor version bump

**Comment by knownasilya** [comment]
No, and I agree on that last point.

**Pull request merged**